config/initializers/session_store.rb

```diff
@@ -2,5 +2,5 @@
 
 Rails.application.config.session_store :cookie_store,
   key: '_publisher_session',
-  secure: Rails.env.production?,
+  secure: Rails.env.production? && !ENV['DISABLE_PROD_SSL'],
```

Comment:
Would it be better to have this as `ENV.fetch("USE_SECURE_COOKIES", Rails.env.production?)` ?

Reply:
We want to be able to specify an ENV var to disable the SSL behaviour.  Given Ruby will cast any string provided in the ENV to a string, we went down this route.

Could do something similar to this:

```ruby
default_secure_cookie = if Rails.env.production? then "yes" else "no" end
use_secure_cookie = ENV.fetch("USE_SECURE_COOKIES", default_secure_cookie) == "yes"
```

Reply:
Ah yes - sorry, my idea was rubbish! 

Your approach seems good - though I guess we might have to add this to loads of repos so I wonder if we can make it more concise. Could be that we went down route of `ENV.fetch("DISABLE_SECURE_COOKIES", !Rails.env.production?)`

Reply:
Think it would need to be ` secure: !ENV.fetch("DISABLE_SECURE_COOKIES", !Rails.env.production?), ` in full.  Is the double negative too jarring?

Reply:
Yeah I'd agree it's a bit jarring. Probably best to just switch back to what it was originally but name it about cookies.